practice-01/styles/variable.css

```diff
@@ -0,0 +1,33 @@
+:root {
+    /* --------------font-------------------- */
+    --font-family: "Satoshi";
+
+    /* -------------color-------------------- */
+    --WHITE: #ffffff;
+    --LIGHTHOUSE: #f4f4f4;
+    --WESTCHESTER-GREY: #787878;
+    --CRAB-NEBULA: #004050;
+
+
+    /* -------------font-weight-------------- */
+    --font-weight-title: 700;
```

Comment:
The same with color. name should be abstract. not for special cases. It's should the same as font-size